Sources/PointFree/Account/Invoices.swift

```diff
@@ -156,9 +156,25 @@ private let invoicesRowView = View<Stripe.ListEnvelope<Stripe.Invoice>> { invoic
   )
 }
 
+private func discountDescription(for discount: Stripe.Subscription.Discount, invoice: Stripe.Invoice) -> String {
+  return "\(format(cents: invoice.total - invoice.subtotal)) (\(discount.coupon.name))"
+}
+
 let invoiceView = View<(Stripe.Subscription, Database.User, Stripe.Invoice)> { subscription, currentUser, invoice -> Node in
 
-  gridRow([
+  let discountRow = invoice.discount.map { discount in
```

Comment:
new row in the invoice. example of what this looks like from screenshot test:

<img width="447" alt="image" src="https://user-images.githubusercontent.com/135203/43610088-55c16ae4-9673-11e8-8be0-7030eb93bb05.png">